test/timeout.test.ts

```diff
@@ -0,0 +1,89 @@
+describe('timeout with abort controller', () => {
+    it('empty', async () => {
+        // empty test so jest won't fail in node < 15 due to no tests being executed
+    });
+
+    if (Number(process.version.match(/v(\d+)/)?.[1] ?? null) < 15) {
```

Comment:
Again, just a small readability nitpick. Personally, I'd rather create an extra variable than do stuff like `)?.[1] ?? null)`. Old school is cool.

```js
const [majorVersion] = process.version.split('.');
if (Number(majorVersion) < 15) {
```

Reply:
You still need to strip the `v` prefix, but sure, we can have some temp var to make it more readable. But as its inside tests I did not care much.

Reply:
`process.versions.node`

Reply:
> But as its inside tests I did not care much.

Ah, right, missed that while skimming the code.

> `process.versions.node`

This is what I meant 👍